src/lib/coda_base/ledger.ml

```diff
@@ -397,21 +397,79 @@ struct
     Account.equal expected_account account
 end
 
-module Ledger = struct
-  include Merkle_ledger.Ledger.Make (Public_key.Compressed) (Account)
-            (struct
-              type t = Ledger_hash.t [@@deriving sexp, hash, compare, bin_io]
+module Depth = struct
+  let depth = ledger_depth
+end
+module Location0 = Merkle_ledger.Location.Make(Depth)
+module In_memory_kvdb : Merkle_ledger.Intf.Key_value_database = struct
```

Comment:
This was copied from elsewhere, I'll unify these before we land this if we think this is a good idea